src/main/webapp/app/shared/layouts/navbar/navbar.component.html

```diff
@@ -198,6 +198,12 @@ <h6 class="dropdown-header fw-medium" jhiTranslate="global.menu.guidedTutorial">
                             <span jhiTranslate="global.menu.admin.privacyStatement">Privacy statement</span>
                         </a>
                     </li>
+                    <li>
+                        <a class="dropdown-item" routerLink="admin/imprint" routerLinkActive="active" (click)="collapseNavbar()">
```

Comment:
```suggestion
                        <a class="dropdown-item" routerLink="/admin/imprint" routerLinkActive="active" (click)="collapseNavbar()">
```
I get this warning in the console:
![screenshot-2023-05-12_001335](https://github.com/ls1intum/Artemis/assets/9006596/0f3bd64f-4380-45d2-b822-10dd2ea50aef)

(same for all other navigation links)
I'm fixing the other links in the navbar in #6546, but this obviously isn't included. Could you change this to `/admin/imprint`?